wazo_dird/plugins/wazo_user_backend/api.yml

```diff
@@ -126,6 +126,7 @@ paths:
         - $ref: '#/parameters/limit'
         - $ref: '#/parameters/offset'
         - $ref: '#/parameters/search'
+        - $ref: '#/parameters/backend_wazo_uuid'
```

Comment:
I think it would be better to call it `wazo_backend_uuid` instead (even though it's only useful in the spec). `backend_wazo_uuid` gives the impression that it is about the Wazo UUID.

Reply:
fixed